### ISSUE TITLE
Can we send custom delimiter as parameter?

### ISSUE THREAD

**Issue by rnjailamba** [victorboissiere/vue-xls-csv-parser#3]

**Comment by victorboissiere** [comment]
Excel should work fine. However, CSV should have some trouble being parsed. Can you upload an example file ? I'll look into it when I have time. Otherwise you can make a pull request in the meantime 😉

**Comment by rnjailamba** [comment]
Yes excel was good, i created a csv parser and csv validator on my own[It can send parsed json to the server or the file as well]. Feel free to copy it if you get time from here https://github.com/rnjailamba/vuehn/blob/master/src/components/Industriesupload.vue ( no time for me as well or i would have done it :) )
